.evergreen.yml

```diff
@@ -124,6 +125,28 @@ tasks:
 
         COVERALLS_REPO_TOKEN=${coveralls_repo_token|} poetry run coveralls
 
+- name: evg_api_install
+  commands:
+  - command: shell.exec
+    params:
+      working_dir: src
+      script: |
+        set -o errexit
+        set -o verbose
+
+        export LC_ALL=C.UTF-8
+        export LANG=C.UTF-8
+
+        mkdir evg-api-cli
+        cd evg-api-cli
+        /opt/mongodbtoolchain/v3/bin/python3 -m venv evg-api-cli-venv
+        . evg-api-cli-venv/bin/activate
+
+        pip install --upgrade pip
+        pip install git+ssh://git@github.com/evergreen-ci/evergreen.py.git
```

Comment:
You will want to perform the pip install from the directory the checked out code is in. Doing the pip install from github will not test the code that has been changed.

```bash
pip install ..
```